### ISSUE TITLE
【SF】【Table】【A11y】More button lacks descriptive accessible name

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#6309]
### Is your feature request related to a problem?

Recently, I've received an a11y issue about the accessible name for more button in `Table`. It will read "More Button" and we can't set the descriptive accessible name for it.
![image](https://github.com/user-attachments/assets/a6402abe-087c-4e48-ab17-6fdb5c369153)

However, the A11y expert expect the similar behavior here. 
![Uploading image.png…]()

### Describe the solution you'd like

Support setting accessible-name for more button in Table.

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @tandan0813 

Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

Please also let my colleagues know which version of the `Table` component you're using. It's coupled with the `@ui5/webcomponents` major version (v1 or v2).
